src/Curl.Converter.Net/Curl.Converter.Net.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
I believe that `netstandard2.1` target should be enough since we don't use any specific .net APIs.

Reply:
You're right. Updated.